web-store/src/main/resources/static/cart.js

```diff
@@ -0,0 +1,109 @@
+var cart = {};
+
+$.getJSON('mig.json', function(data) {
+    chekCart();
+    showCart();
+
+    function showCart() {
```

Comment:
I tried but everything went crash

Reply:
Because you cart variable is global. It is bad practice.
Please define showCartItem() and execute it where it is needed
```

$('document').ready(function () {
    showCartItem()
})

function showCartItem() {
    $.getJSON('mig.json', function (data) {
        showCartData(data);
    })
}
```